macros/form.njk

```diff
@@ -125,15 +125,14 @@
   {{ option(label, name, form, value, 'radio', hint, checked, bold) }}
 {% endmacro %}
 
-{% macro checkboxOption(label, name, form, value, hint = '', checked = false, bold = false) %}
-  {{ option(label, name, form, value, 'checkbox', hint, checked, bold) }}
+{% macro checkboxOption(label, name, form, value, hint = '', checked = false, bold = false, disabled = false) %}
```

Comment:
are we sure we want this?
http://govuk-elements.herokuapp.com/buttons/#button-disabled

Reply:
if we do it, I think it should be a separate macro, with some docs on it to show that in general you should not be doing it